src/timer/timer_manager.h

```diff
@@ -77,9 +77,12 @@ class TimerManager {
 	 * The implementation per type is different, but they all share a similar goal:
 	 *   Call the Elapsed() method of all active timers.
 	 *
+	 * There is an optional return value that timers may use to signal if they have "ticked".
+	 *
 	 * @param value The amount of time that has elapsed.
+	 * @return A boolean value to signal that something has "ticked," not used by all timers.
```

Comment:
Here too, I would make it simpler: `True iff time has progressed`.

Btw, `a boolean value` is totally wasting bytes. That it is a boolean, we can see, and that it is a value, is obvious :)